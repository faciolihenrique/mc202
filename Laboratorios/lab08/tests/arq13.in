# Teste com alunos da Turma E em ordem decrescente de RA
i 160245 Rogerio Guimaraes Meirelles
i 160160 Guilherme Furlan
i 160138 Caio Cesar de Campos Silva
i 158526 Yuri Catarino Delgado
i 158372 Raphael Pavan Silva
i 157240 Sabrina Beck Angelini
i 157228 Ronaldo Prata Amorim
i 157198 Rodrigo Dahlstrom Hilkner
i 156405 Lucas Rodolfo de Castro Moura
i 156131 Lais Tiemi Murakami
i 156074 Kaique de Souza Monteiro
i 155951 Joao Vitor Buscatto Silva
i 155925 Joao Pedro Brasao Toledo
i 155758 Igor Gustavo Hitzschky Lema
i 155742 Hugo Ralf Pereira
i 154815 Breno Pedroni
i 149332 Mateus Roque Buck
i 146328 Guilherme Victor Cunha Câmara
i 139240 Vítor Monteiro Francisco dos Santos
i 137796 Victor Otávio Andrade das Neves
i 117706 Lucas Cavalcante de Barros
i 105351 Maria Angelica de Araujo Barnabe
i 104923 Fernando Henrique de Moura Bastos
i 103179 Lucas Seiji Mizuta
i 043473 Fernanda Elena de Barros Reis
p
n
a
d
l
x
