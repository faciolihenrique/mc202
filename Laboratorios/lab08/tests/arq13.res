Ação: # Teste com alunos da Turma E em ordem decrescente de RA
Ação: i 160245 Rogerio Guimaraes Meirelles
Ação: i 160160 Guilherme Furlan
Ação: i 160138 Caio Cesar de Campos Silva
Ação: i 158526 Yuri Catarino Delgado
Ação: i 158372 Raphael Pavan Silva
Ação: i 157240 Sabrina Beck Angelini
Ação: i 157228 Ronaldo Prata Amorim
Ação: i 157198 Rodrigo Dahlstrom Hilkner
Ação: i 156405 Lucas Rodolfo de Castro Moura
Ação: i 156131 Lais Tiemi Murakami
Ação: i 156074 Kaique de Souza Monteiro
Ação: i 155951 Joao Vitor Buscatto Silva
Ação: i 155925 Joao Pedro Brasao Toledo
Ação: i 155758 Igor Gustavo Hitzschky Lema
Ação: i 155742 Hugo Ralf Pereira
Ação: i 154815 Breno Pedroni
Ação: i 149332 Mateus Roque Buck
Ação: i 146328 Guilherme Victor Cunha Câmara
Ação: i 139240 Vítor Monteiro Francisco dos Santos
Ação: i 137796 Victor Otávio Andrade das Neves
Ação: i 117706 Lucas Cavalcante de Barros
Ação: i 105351 Maria Angelica de Araujo Barnabe
Ação: i 104923 Fernando Henrique de Moura Bastos
Ação: i 103179 Lucas Seiji Mizuta
Ação: i 043473 Fernanda Elena de Barros Reis
Ação: p
043473 Fernanda Elena de Barros Reis
103179 Lucas Seiji Mizuta
104923 Fernando Henrique de Moura Bastos
105351 Maria Angelica de Araujo Barnabe
117706 Lucas Cavalcante de Barros
137796 Victor Otávio Andrade das Neves
139240 Vítor Monteiro Francisco dos Santos
146328 Guilherme Victor Cunha Câmara
149332 Mateus Roque Buck
154815 Breno Pedroni
155742 Hugo Ralf Pereira
155758 Igor Gustavo Hitzschky Lema
155925 Joao Pedro Brasao Toledo
155951 Joao Vitor Buscatto Silva
156074 Kaique de Souza Monteiro
156131 Lais Tiemi Murakami
156405 Lucas Rodolfo de Castro Moura
157198 Rodrigo Dahlstrom Hilkner
157228 Ronaldo Prata Amorim
157240 Sabrina Beck Angelini
158372 Raphael Pavan Silva
158526 Yuri Catarino Delgado
160138 Caio Cesar de Campos Silva
160160 Guilherme Furlan
160245 Rogerio Guimaraes Meirelles
Ação: n
Número de nós na base de dados: 25.
Ação: a
Altura da base de dados: 5.
Ação: d

      /160245
     /160160
      \160138
    /158526
      /158372
     \157240
      \157228
   /157198
      /156405
     /156131
      \156074
    \155951
      /155925
     \155758
      \155742
 154815
     /149332
    /146328
     \139240
   \137796
     /117706
    \105351
      /104923
     \103179
      \ 43473

Ação: l
Ação: x
Processamento terminado
