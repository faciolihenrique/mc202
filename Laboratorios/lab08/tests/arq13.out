Ação: # Teste com alunos da Turma E em ordem decrescente de RA
Ação: i 160245 Rogerio Guimaraes Meirelles
Ação: i 160160 Guilherme Furlan
Ação: i 160138 Caio Cesar de Campos Silva
